File: BEE 1072.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    int n;
    cin >> n;
    int cnt = 0;
    for (int i = 0; i < n; i++){
        int num;
        cin >> num;
        if (num >= 10 && num <= 20){
            cnt += 1;
        }
    }
    cout << cnt << " in" << endl;
    cout << (n - cnt) << " out" << endl;
    return 0;
}

File: BEE 1072.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

var cnt = 0;

for (var i = 0; i < n; i++){
    var num = parseInt(lines.shift());
    if (num >= 10 && num <= 20){
        cnt += 1;
    }
}

console.log(cnt +" in");
console.log((n - cnt) +" out");